pkg/types/types.go

```diff
@@ -188,6 +186,15 @@ type Fault struct {
 	FailAt string `json:"failAt,omitempty"`
 }
 
+// MustBytes returns an encoded json representation of the condition or panics
+func (c *Condition) MustBytes() []byte {
+	byt, err := json.Marshal(c)
+	if err != nil {
+		panic("encoding condition failed: " + err.Error())
```

Comment:
I'm not sure what you mean? This provides some syntactic sugar to avoid the clutter of dealing with a "should never happen" error every time we serialize a condition.